nextflow_schema.json

```diff
@@ -587,7 +608,6 @@
                 "bt2n": {
                     "type": "integer",
                     "description": "Specify the -N parameter for bowtie2 (mismatches in seed). This will override defaults from alignmode/sensitivity.",
-                    "default": 0,
```

Comment:
Why are you removing the `default` fields?

Reply:
I _think_ this was something happening with the old nf-core/tools 1.11/1.12 release(s).

Reply:
I would say keep them in then 😉

Reply:
Yeah, re-added!